tests/filecheck/backend/riscv/memref_to_riscv_opt.mlir

```diff
@@ -25,9 +25,9 @@ builtin.module {
 // CHECK:      builtin.module {
 // CHECK-NEXT:   %vf_reg, %vi_reg, %mf_reg, %mi_reg = "test.op"() : () -> (!riscv.freg<>, !riscv.reg<>, !riscv.reg<>, !riscv.reg<>)
 // CHECK-NEXT:   riscv.fsw %mf_reg, %vf_reg, 12 {"comment" = "store float value to memref of shape (3, 2)"} : (!riscv.reg<>, !riscv.freg<>) -> ()
-// CHECK-NEXT:   %xf = riscv.flw %mf_reg, 12 {"comment" = "load value from memref of shape (3, 2)"} : (!riscv.reg<>) -> !riscv.freg<>
+// CHECK-NEXT:   %xf = riscv.flw %mf_reg, 12 {"comment" = "load float from memref of shape (3, 2)"} : (!riscv.reg<>) -> !riscv.freg<>
```

Comment:
There's no f64 test in this example?

Reply:
No, I'll add it. I have a feeling there are a few canonicalization patterns missing for it to work, but that'll be a good thing to track.

Reply:
Turns out it just works, updated